workspaces/tests/account.rs

```diff
@@ -8,7 +8,7 @@ use std::path::Path;
 
 #[test(tokio::test)]
 async fn test_subaccount_creation() -> anyhow::Result<()> {
-    let worker = near_workspaces::sandbox().await?;
```

Comment:
Testing with testnet will drain the faucet too fast, unless we delete the accounts and return all unused tokens back to the faucet #215.

Either resolve #215 (at least manually deleting the accounts in the test) or revert it back to `sandbox`

Reply:
I think this is still actual.
Even though, we don't use testnet in tests. I think we should motivate users to delete and remove created accounts.

Maybe, we should have something as `Drop` implementation that would `panic` if user is not deleted. 
Just thinking loud :)

Reply:
i don't think this is a real issue of users, who just want to routinely do their integration tests and get some test NEAR in the process. 
this is an issue of users, who intentionally want to drain faucet just for fun or for some other reason, and they still can do it via other means, outside of `near-workspaces-rs` api.